src/test/java/PetTest.java

```diff
@@ -0,0 +1,118 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PetTest {
+
+    @BeforeEach
```

Comment:
ik heb hier name toegevoegd zodat ik dat niet bij elke test opnieuw hoeft te schrijven: bijvoorbeeld
me = new Person("Kay-Arne","Fratheo","Heskes",31,'m');

Reply:
in jou geval hoef je dus maar 1 keer Pet Alma hierin verwerken